### DISCUSSION TITLE
[Request] Temporary softmuting

### DISCUSSION THREAD

**Discussion by v8mins** [GriefPrevention/GriefPrevention#289] [Ideas]
Hey there,

Would be neat if you could make it so that softmutes can be contingent to a time specification, like ordinary mutes, e.g /softmute (player) 10m

**Comment by ghost** [comment]
I like this idea too... spam mutes could be configured to be more aggressive as well, for short duration mutes. This could help some servers, a custom anti-spam setup could be made or something.

**Comment by TomLewis** [comment]
I forget that softmute is a thing and have to clean out the softmute file every 6 months or so, last time. Had over 300 people soft muted, I might just disable it and keep to chatcontrol pro.

**Comment by RoboMWM** [comment]
@UoibiE Labels are a useful thing
